dspace-api/src/main/java/org/dspace/app/itemimport/ItemImportCLITool.java

```diff
@@ -94,6 +94,7 @@ public static void main(String[] argv) throws Exception {
             options.addOption("R", "resume", false,
                               "resume a failed import (add only)");
             options.addOption("q", "quiet", false, "don't display metadata");
+            options.addOption("l", "relationships", false, "process relationships manifest");
```

Comment:
Since this option only works for "add", we should note that in this description.  It could simply be updated to `"process relationships manifest (add only)"`

Reply:
Gotcha. I was also wondering whether it should be enabled by default, since it will not do anything if there is no relationships manifest?

Reply:
@tysonlt: Yes, that's a reasonable option to just enable this at all times.  If the `relationships` manifest isn't there, it should do nothing.  Though it also probably should be a little less "noisy" then...it can say it's "checking for relationships", but likely should then only report on which items it finds a `relationships` manifest for (and not items it finds nothing for).   

This sort of behavior could allow us to simply remove the `-l` flag altogether.  It's just always running, but it doesn't say much if no `relationships` manifests are found anywhere.